tests/prototest/test.golden

```diff
@@ -51,6 +51,9 @@ table ProtoMessage {
   /// doc comment for r.
   r:proto.test.ProtoMessage_.Anonymous0;
   outer_enum:proto.test.ProtoEnum;
+  u:float = +inf;
+  v:float = +inf;
```

Comment:
did you mean one of these two to be plain `inf` without `+` ?

Reply:
No, I wanted it to be explicitly signed, even if the input is unspecified.